HW_3/HW_3_Task_2.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+// Task 2*
+// Напишите программу, которая принимает целое положительное число n (одно любое число от 1 до 9), 
+// и выводит сумму равную n + nn + nnn, где n не перемножаются, а конкатенируются.
+
+// Вариант 1:
+let n;
```

Comment:
Значение присвоить тоже нужно, а то в консоле `NaN`
Используем `const` где можем
```suggestion
const n = 1;
```